docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -336,6 +336,30 @@ Given multipart request:
 |binary|binary-key|raw              |text/plain |raw.txt        |
 ----
 
+=== Set form data HTTP request
+
+Sets https://www.w3.org/TR/html401/interact/forms.html#h-17.13.4.1[urlencoded] request entity that will be used while executing HTTP requests.
+Default HTTP request header with name `Content-Type` and value `application/x-www-form-urlencoded; charset=UTF-8` is set.
+In case if `Content-Type` is `text/plain` <<Set HTTP request body>> should be used
```

Comment:
```
<<Set HTTP request body>>
```
 - does it work?

Reply:
do you mean 'link to Set HTTP request body section' or method 'Set HTTP request body' in general?